src/main/java/com/spring/board/springboard/post/domain/dto/RequestPostDTO.java

```diff
@@ -0,0 +1,48 @@
+package com.spring.board.springboard.post.domain.dto;
+
+import com.spring.board.springboard.post.domain.Post;
+import com.spring.board.springboard.user.domain.Member;
+
+import java.time.LocalDateTime;
+
+public class RequestPostDTO {
+    private String title;
+    private String content;
+    private Integer memberId;
+
+    public RequestPostDTO() {
+    }
+
+    public RequestPostDTO(String title, String content, Integer memberId) {
+        this.title = title;
+        this.content = content;
+        this.memberId = memberId;
+    }
+
+    public RequestPostDTO(Post post) {
+        this.title = post.getTitle();
+        this.content = post.getContent();
+        this.memberId = post.getMemberId();
+    }
+
+    public Post toEntity(RequestPostDTO postDTO, Member member) {
+        return new Post(
+                postDTO.getTitle(),
+                postDTO.getContent(),
+                LocalDateTime.now(),
+                member
+        );
+    }
```

Comment:
자기 자신을 인자로 받을 필요는 없을 것 같네요
```suggestion
    public Post toEntity(Member member) {
        return new Post(
                getTitle(),
                getContent(),
                LocalDateTime.now(),
                member
        );
    }
```